Engine/source/math/mMatrix.h

```diff
@@ -212,7 +214,23 @@ class MatrixF
    /// Convenience function to allow people to treat this like an array.
    F32& operator ()(S32 row, S32 col) { return m[idx(col,row)]; }
    F32 operator ()(S32 row, S32 col) const { return m[idx(col,row)]; }
-
+   
+   bool operator==(const MatrixF& a) const
+   {
+      return (a[0] == m[0] && a[1] == m[1] && a[2] == m[2] && a[3] == m[3]
```

Comment:
I wouldn't. memcmp won't give the right answers for floats, e.g. 0.0 wouldn't compare equal to -0.0 as it should. Guessing it would be slower to call memcmp too.